go/service/main.go

```diff
@@ -242,6 +248,13 @@ func (d *Service) Run() (err error) {
 	return err
 }
 
+func (d *Service) setupGlobals() error {
+	g := d.G()
+	teams.NewTeamLoaderAndInstall(g)
+	pvlsource.NewPvlSourceAndInstall(g)
```

Comment:
this seems like an odd pair to be the two "Globals" for the service...pvl and teams loader.

I understand you moved these functions from elsewhere, but `setupGlobals` sounds like it's doing a lot more than constructing these two things.